This patch is against the installed OOo and should be applied to
<OOO_HOME>/share/psprint/psprint.conf

--- share/psprint/psprint.conf.old	2004-11-09 17:38:35.000000000 +0100
+++ share/psprint/psprint.conf	2006-05-19 16:17:07.000000000 +0200
@@ -70,7 +70,7 @@
 Printer=SGENPRT/Generic Printer
 
 ; DefaultPrinter: marks the default printer
-DefaultPrinter=1
+DefaultPrinter=0
 
 ; Location: a user readable string that will be shown in the print dialog
 Location=
@@ -96,3 +96,29 @@
 SubstFont_Thorndale=Times
 SubstFont_Timmons=Times
 SubstFont_Cumberland=Courier
+
+[PDF converter]
+Printer=SGENPRT/PDF converter
+DefaultPrinter=1
+Location=
+Comment=
+Command=/usr/bin/gs -q -dNOPAUSE -sDEVICE=pdfwrite -sOutputFile="(OUTFILE)" -
+Features=pdf=/tmp
+Copies=1
+Scale=100
+Orientation=Portrait
+PSLevel=0
+ColorDevice=0
+ColorDepth=24
+MarginAdjust=0,0,0,0
+PPD_PageSize=Letter
+PerformFontSubstitution=true
+SubstFont_Arial=Helvetica
+SubstFont_Times New Roman=Times
+SubstFont_Cumberland=Courier
+SubstFont_Timmons=Times
+SubstFont_Helmet=Helvetica
+SubstFont_Thorndale=Times
+SubstFont_Albany=Helvetica
+SubstFont_Courier New=Courier
+
